app/models/map/presenter.rb

```
@@ -45,7 +45,7 @@ def to_poro
 
       def bounds_from(map)
         ::JSON.parse("[#{map.view_bounds_sw}, #{map.view_bounds_ne}]")
-      rescue => exception
+      rescue #=> exception
```

I'd, at least, send error to Rollbar, there's no good reason for swallowing errors, and add a comment explaining why is it done.